packages/core/src/customizations/fragments/pdp.ts

```diff
@@ -0,0 +1,18 @@
+import { gql } from '@faststore/graphql-utils'
+
+export const fragment = gql`
+  fragment pdp on Query {
+    product(locator: [{ key: "slug", value: $slug }]) {
+      id: productID
+      ...CustomProductDetailsFragment_StoreProduct
```

Comment:
default pdp fragment simulating the one that comes from starter already with the `CustomProductDetailsFragment_StoreProduct` defined by the custom section.